src/js/pages/product.js

```diff
@@ -0,0 +1,31 @@
+export function loadData() {
+  var productApi = "http://localhost:4000/products";
+
+  fetch(productApi)
+    .then((response) => response.json())
+    .then((data) => {
+      var productGrid = document.getElementsByClassName('product-grid')[0];
+      data.map((product) => {
```

Comment:
```suggestion
    .then((products) => {
      const productGrid = document.querySelector('.product-grid');
      
      products.map((product) => {
```
Sử dụng querySelector cho đơn giản hơn nhé